client/src/store/selectors.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright © 2022-2024, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export { logSelectors } from "./log/selectors";
+// export { runSelectors } from "./run/selectors";
```

Comment:
Looks like we can clean this up for now

Reply:
Fixed in https://github.com/sassoftware/vscode-sas-extension/pull/713/commits/b53a6ba752b9c595d468a795ee1a79100fc84c3c.